airbyte-webapp/src/components/connection/ConnectionOnboarding/ConnectionOnboarding.tsx

```diff
@@ -0,0 +1,256 @@
+              </button>
+            }
+          >
+            {moreSourcesTooltip}
+          </Tooltip>
+        </div>
+        <div className={styles.airbyte} aria-hidden="true">
+          <AirbyteIllustration sourceHighlighted={highlightedSource} destinationHighlighted={highlightedDestination} />
+        </div>
+        <div className={styles.destinations}>
+          <Text bold as="div" className={styles.destinationsTitle}>
+            <Tooltip
+              control={
+                <span>
+                  <FormattedMessage id="connection.onboarding.destinations" />{" "}
+                  <FontAwesomeIcon icon={faCircleQuestion} />
+                </span>
+              }
+            >
+              <FormattedMessage id="connection.onboarding.destinationsDescription" />
+            </Tooltip>
+          </Text>
+          {destinations.map((destination, index) => {
+            const tooltipText = formatMessage(
+              { id: "connection.onboarding.addDestination" },
+              { destination: destination?.name }
+            );
+            return (
+              <Tooltip
+                key={destination?.destinationDefinitionId}
+                placement="right"
+                control={
+                  <button
+                    className={styles.connectorButton}
+                    // onMouseEnter doesn't trigger on disabled buttons in React
+                    // https://github.com/facebook/react/issues/10109
+                    // Thus we just disable it via aria-disabled and make it non focusable via tabindex
+                    onMouseEnter={() => setHighlightedDestination(index as 0 | 1 | 2 | 3)}
+                    aria-disabled="true"
+                    aria-label={tooltipText}
+                    tabIndex={-1}
+                  >
+                    <div className={styles.connectorIcon}>{getIcon(destination?.icon)}</div>
+                  </button>
+                }
+              >
+                {tooltipText}
+              </Tooltip>
+            );
+          })}
+          <Tooltip
+            placement="right"
+            control={
+              <button
+                className={styles.connectorButton}
+                // onMouseEnter doesn't trigger on disabled buttons in React
+                // https://github.com/facebook/react/issues/10109
+                // Thus we just disable it via aria-disabled and make it non focusable via tabindex
+                onMouseEnter={() => setHighlightedDestination(3)}
+                aria-disabled="true"
+                aria-label={moreDestinationsTooltip}
+                tabIndex={-1}
+              >
+                <PlusIcon className={styles.moreIcon} />
+              </button>
+            }
+          >
+            {moreDestinationsTooltip}
+          </Tooltip>
+        </div>
+      </div>
+      <div className={styles.footer}>
+        <Button onClick={() => onCreate()} size="lg" data-id="new-connection">
```

Comment:
```suggestion
        <Button onClick={onCreate} size="lg" data-id="new-connection">
```

Reply:
This would not compile since it breaks types. `onCreate` can accept a parameter (which source to load), so we can't pass the ClickEvent into this.